cmd/wallet-web/src/components/InputField/InputField.vue

```diff
@@ -1,23 +1,22 @@
-<!--
- * Copyright SecureKey Technologies Inc. All Rights Reserved.
- *
- * SPDX-License-Identifier: Apache-2.0
--->
+/* Copyright SecureKey Technologies Inc. All Rights Reserved. SPDX-License-Identifier: Apache-2.0 */
 
 <template>
   <div class="input-container">
     <input
       :id="'input-' + label"
       v-bind="$attrs"
-      :value="value"
+      :disabled="isDisabled"
       v-on="$listeners"
       @input="$emit('update', $event.target.value)"
+      @keyup="remaincharCount()"
     />
     <label :for="'input-' + label" class="input-label">{{ label }}</label>
     <span class="input-helper">{{ helperMessage }}</span>
-    <div class="fader" />
-    <!-- TODO: use inline svg instead once https://github.com/trustbloc/edge-agent/issues/816 is fixed -->
-    <img src="@/assets/img/danger-icon.svg" />
+    <div class="fader">
+      <!-- TODO: use inline svg instead once https://github.com/trustbloc/edge-agent/issues/816 is fixed -->
+      <img src="@/assets/img/danger-icon.svg" />
+    </div>
```

Comment:
This change does not seem to be correct. The danger icon is not supposed to be placed on the fader, it's supposed to be to the right from it.

Reply:
fixed as original